ui/apps/platform/cypress/helpers/main.js

```diff
@@ -26,7 +26,7 @@ export function visitMainDashboard(requestConfig, staticResponseMap) {
 
 // TODO Make this the default once phase one of the PF Dashboard is enabled
 export function visitMainDashboardPF() {
-    visit(pfUrl);
+    visit(url);
```

Comment:
What are pro and con instead to delete this function and replace it with `visitMainDashboard` in cypress/integration/dashboard/dashboardSecurityMetrics.test.js to ensure single source for all helpers and tests which also call it?

Reply:
That was what I went with first, but the wait in `visitMainDashboard` was conflicting with a wait in the individual e2e tests for the new dashboard. The request we are awaiting in the individual tests would fire and resolve before the `deploymentsWithProcessInfo` one and cause a failure.

I went with this as a less-intrusive way to enable the feature flag without (hopefully) causing new flakes before the release. My plan is to replace this test with jest/testing-library tests once the dust settles. The original awaited network call should work fine for other tests that rely on `visitMainDashboard` since both the old dashboard and the new one make that request.

Reply:
Aha, that makes sense.